install

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+#TODO(by black7375)
+#However, there may be existing user's settings, so back up and make a copy.
+#I just made a simple one yesterday, assuming that there may be multiple backup states.
+#I think it's better to do a redirection(>>) after backup to merge the contents of the file.
+printf "Are you using ESR, Default, or Dev?(ESR/Default/Dev): "
+read -r
+if [ "$REPLY" == "ESR" ] || [ "$REPLY" == "esr" ]; then
+  profiledir="$(grep '.default-esr' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
+elif [ "$REPLY" == "Default" ] || [ "$REPLY" == "default" ]; then
+  profiledir="$(grep '.default-release' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
+elif [ "$REPLY" == "Dev" ] || [ "$REPLY" == "dev" ]; then
+  profiledir="$(grep '.dev-edition-default' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
+else
+  echo "Unspecified."
+  exit
+fi
+if [[ -f ~/.mozilla/firefox/${profiledir}/user.js ]]; then
+    printf "user.js exists. Do you want to make a backup of it?(Y/n): "
+    read -r
+    if [ "N" == "$REPLY" ] || [ "n" == "$REPLY" ]; then
+      printf "Overwriting...\n"
```

Comment:
Repeat Code2

No need to overwite and backup repeatedly. (`Line 40`)
Define and use actions as function.

Reply:
ok.